workflow.md

```diff
@@ -0,0 +1,99 @@
+# An Introduction to the CLEX Research Workflow
+
+Below we describe 2 standard workflows that researchers might follow:
+
+1. Typical for users of NCI's High Performance Computing (HPC) resources
+2. Typical for non-NCI users
+
+For NCI Users
+-------------
+
+<details><summary>NCI Induction</summary>
+  
+  * Joining a project
+  * Command line basics - bash
+  * File systems
+  
+</details>
+<details><summary>Planning resource needs</summary>
+
+  * CPU
+  * Storage
+  * Ongoing management
+</details>
+
+<details><summary>Finding data to run the model on (or to analyse)</summary>
+
+  * Clef
+  * NCI data catalogue
+  * Geonetwork
+  * THREDD link
+</details>
+
+<details><summary>Running a model</summary>
+
+  * Configuring and running the model
+    * MOM - Payu
+    * The job queue
+  * Postprocessing
+</details>
+
+<details><summary>Analysing model results (or any data)</summary>
+
+  * VDI
+  * The job queue
+  * Xarray
+  * Plotting
+</details>
+
+<details><summary>Code development and management</summary>
+
+  * Fortran basics
+  * Python basics
+  * Debugging
+  * Version control
+  * Svn
+  * Git
+  * Publishing
+</details>
+
+<details><summary>Publishing data</summary>
+
+  * ???
+</details>
+
+For Non-NCI Users
+-----------------
+<details><summary>Finding data to run the model on (or to analyse)</summary>
+
+  * Clef
+  * NCI data catalogue
+  * Geonetwork
+  * THREDD link
+</details>
+
+<details><summary>Running a model locally</summary>
+  * ???
```

Comment:
We can't support any code that isn't run on raijin. Whether we should be encouraging/suggesting it is another issue. Not sure

Reply:
The idea of having the non-NCI workflow is not so that we should support it, but that so non-NCI users who look at the NCI list don't think they don't need any of those skills. But I take your point that we don't want to send the wrong message.